recipe/meta.yaml

```
@@ -1,45 +1,60 @@
 {% set name = "posthog" %}
-{% set version = "2.2.0" %}
+{% set version = "3.16.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1f450a20f7bed217874ded234f6ffc24ae73da37a16aa5bf76335249b34a9f60
+  sha256: 953176a443b30b1404c0f36010a95caad60a83c31ecb17b427f6d986f6f765c1
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.7.0,<4.0.0
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7.0,<4.0.0
+    - python
     - requests >=2.7.0,<3.0.0
-    - six >=1.5.0,<2.0.0
-    - monotonic >=1.5.0,<2.0.0
-    - backoff >=1.10.0,<2.0.0
-    - python-dateutil >=2.1.0,<3.0.0
+    - six >=1.5.0
+    - monotonic >=1.5.0
+    - backoff >=1.10.0
+    - python-dateutil >=2.1.0
```

```suggestion
    - python-dateutil >2.1.0
```
see https://github.com/PostHog/posthog-python/blob/v3.16.0/setup.py#L22